src/QuoteBox.js

```diff
@@ -2,12 +2,12 @@ import React, { useState, useEffect } from "react";
 import axios from "axios";
 import "./styles.scss";
 
+const api_url = "https://api.quotable.io/random";
+
 const QuoteBox = () => {
   const [quote, setQuote] = useState("...Ops, something went wrong");
```

Comment:
this will mean that you are setting error as the initial state before anything is called at all hence the user gets to see error message even when no response of error